captum/insights/frontend/src/components/FilterContainer.tsx

```diff
@@ -64,18 +76,18 @@ class FilterContainer extends React.Component {
     this.setState({ method_arguments });
   };
 
-  handleSubmit = (event) => {
+  handleSubmit = (event: React.FormEvent) => {
     const method = this.state.selected_method;
     const method_arguments = this.state.method_arguments;
     const argument_config =
       method in method_arguments ? method_arguments[method] : {};
-    const args = {};
+    const args: any = {};
```

Comment:
should this type be `{ [key: string]: MethodsArguments }`

Reply:
Ideally it should be `{ [key: string]: string | boolean | number }` as its just key value pair. Will update it.